doc/source/tune/api_docs/suggestion.rst

```diff
@@ -1,73 +1,195 @@
-.. _searchalg-ref:
+.. _tune-search-alg:
 
 Search Algorithms (tune.suggest)
 ================================
 
-.. _repeater-doc:
+Tune's Search Algorithms are wrappers around open-source optimization libraries for efficient hyperparameter selection. These integrations optimize the given trainable over a provided search space. You can utilize these search algorithms as follows:
+
```

Comment:
can you give a warning that each one is implemented differently or whatever we chatted about earlier today?

Reply:
done